packages/react-icons/src/createIcon.tsx

```diff
@@ -1,4 +1,4 @@
-import * as React from 'react';
+import React from 'react';
```

Comment:
What changes here?

Reply:
We don't need the [importStar helper](https://github.com/microsoft/tslib/blob/master/tslib.js#L227) in our generated code anymore.